src/snowflake/snowpark/session.py

```diff
@@ -361,6 +362,11 @@ def __init__(
         self._analyzer = (
             Analyzer(self) if isinstance(conn, ServerConnection) else MockAnalyzer(self)
         )
+        self._plan_builder = (
```

Comment:
Let's use the plan_builder in `self._analyzer`.

Reply:
good suggestion! I updated to mock_analyzer to create mock builder and used it